cognite/powerops/preprocessor/utils.py

```diff
@@ -43,22 +43,35 @@ def shift_datetime_str(dt: str, days: int) -> str:
 
 
 def initialize_cognite_client() -> CogniteClient:
-    logger.info("Setting up CogniteClient...")
     os.environ["COGNITE_DISABLE_PYPI_VERSION_CHECK"] = "true"
 
-    token = os.getenv("TOKEN")
     project = os.getenv("PROJECT")
     base_url = os.getenv("BASE_URL") or "https://api.cognitedata.com"
 
-    client = CogniteClient(
-        config=ClientConfig(
-            client_name="CogShop-local-debug",
-            base_url=base_url,
-            project=project,
-            credentials=Token(token),
-            timeout=90,
+    logger.info(f"Setting up CogniteClient towards project {project}...")
+
+    if token := os.getenv("TOKEN"):
```

Comment:
In which cases are a token used?

Reply:
it was there already - should I remove it? I'm just using client credentials